### PR TITLE
Sanctuary (Cathedral F5)

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -3908,7 +3908,7 @@ QUEST_LV_0200_20150323_003907	I would say that was the wisdom of Maven.{nl}Maven
 QUEST_LV_0200_20150323_003908	So if we could just recharge the magical power, any kinds of magics will be okay.{nl}Even if that's a magical power from the demons.
 QUEST_LV_0200_20150323_003909	Go to Veiniki altar when you recharge the sacred symbol of the spiritual power.{nl}The third key is waiting for you.
 QUEST_LV_0200_20150323_003910	{memo X}If you think you are ready, I will show you the true nature of the curse.{nl}If you can't eliminate it at once, it will make a bigger trouble.
-QUEST_LV_0200_20150323_003911	So now, the last verification is left.{nl}The scripture that will possess my spirit, Please open it in front of the altar of the revelation.
+QUEST_LV_0200_20150323_003911	$Now all that's left is the Verification Test.{nl}The scripture that possesses my spirit, please open it in front of the altar of the revelation.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3911]: Verification Test? Is it an item? Is it a proper noun?

Comment (vtange) [inline QUEST_LV_0200.tsv:3911]: Aurelius and the Cathedral treat it like a formality. You basically duel with a 'champion of the gate' type figure and after beating him to 10% hp the gate opens.

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -3368,12 +3368,12 @@ QUEST_LV_0200_20150323_003367	I heard the story of the disciple who succeded rem
 QUEST_LV_0200_20150323_003368	The special bishop is at the Great Cathedral.{nl}He died when he was young, but he still stays at the Great Cathedral because of his special mission.
 QUEST_LV_0200_20150323_003369	I tried my best to hide Maven's secret, but..{nl}Naktis is also staying at the Great Cathedral too long.
 QUEST_LV_0200_20150323_003370	Disciple Goda
-QUEST_LV_0200_20150323_003371	The Great Cathedral has collapsed, but it still keeps its beauty.{nl}Especially, the secrets of Maven, the great bishop, looks amazing everytime we see them.
-QUEST_LV_0200_20150323_003372	Maven, the great bishop, contributed everything builiding this Great Cathedral.{nl}All that was collapsed on Medis Diena.. destiny is too harsh.
-QUEST_LV_0200_20150323_003373	Farewell is short, but the relationship lasts long.{nl}Someday we will meet again in the Goddess.
+QUEST_LV_0200_20150323_003371	$The Great Cathedral has collapsed, but it still keeps its beauty.{nl}Especially, the secrets of Maven, the great bishops, they look amazing every time we see them.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3371]: Especially, the secrets of Maven, the great bishops -> Especially, the secrets of Archbishop Maven

(This should be changed in your other lines as well.)

Comment (vtange) [inline QUEST_LV_0200.tsv:3371]: ah, i was slightly confused then but it's clear now. Wow, what word ordering can do to you

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -9386,7 +9386,7 @@ QUEST_LV_0200_20151001_009389	I just hope the chasers won't find this place.
 QUEST_LV_0200_20151001_009390	We can't trust the demons anymore...
 QUEST_LV_0200_20151001_009391	You better get the black herbs fast.
 QUEST_LV_0200_20151001_009392	The disciples are only concentrating on the altars so they don't care whether we die in hunger or not.{nl}They think everything will be solved if they pray hard.{nl}That's why we decided to take an action.
-QUEST_LV_0200_20151001_009393	You will pay for your intent of obtaining the revelation.{nl}Of course, you should obtain the fourth key of Maven as well.
+QUEST_LV_0200_20151001_009393	$It is most definitely sinful to obtain the revelation in this manner.{nl}But of course, this is the only way to obtain Maven's Fourth Key.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:9389]: Fourth Key -> fourth key

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**